services/account-without-juggler/index.ts

```diff
@@ -0,0 +1,39 @@
+import { Application, Server } from 'loopback-next/packages/core';
+import { AccountController } from './controllers/AccountController';
+import { accountRepository } from './repositories/account';
+
+class AccountMicroservice extends Application {
+  private _startTime: Date;
+
+  constructor() {
+    super();
+    const app = this;
+    app.controller(AccountController);
+    app.bind('repositories.account').toClass(accountRepository);
```

Comment:
It would be nice to introduce `app.repository` API to take care of this.

```ts
app.repository(accountRepository);
```

The argument should be strongly typed to a base Repository interface, which is something that we cannot achieve via `bind().toClass()`, at least not now.